src/test/common/TestdataGenerator.kt

```diff
@@ -572,6 +580,38 @@ fun lagPersonResultat(
     return personResultat
 }
 
+fun lagTilkjentYtelse(utbetalingsoppdrag: Utbetalingsoppdrag, behandling: Behandling) =
+    TilkjentYtelse(
+        utbetalingsoppdrag = objectMapper.writeValueAsString(utbetalingsoppdrag),
+        behandling = behandling,
+        opprettetDato = LocalDate.now(),
+        endretDato = LocalDate.now()
+    )
+
+fun lagUtbetalingsoppdrag(utbetalingsperiode: List<Utbetalingsperiode>) = Utbetalingsoppdrag(
+    kodeEndring = Utbetalingsoppdrag.KodeEndring.NY,
+    fagSystem = "BA",
```

Comment:
```suggestion
    fagSystem = "KS",
```
Det har ikke så veldig mye å si gitt at det er testdata, men noen ganger sjekker jeg testdata for å se hvordan ting hadde sett ut i preprod/prod, så hadde vært fint hvis det hadde stemt overrens med KS verdier her 🙌